_posts/2022-04-19-debt-reduction-in-the-product-roadmap.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: "How to include refactoring into Product development roadmap? Reducing technical debt inspired by real options identification"
+author: [olga.dudzik]
+tags: [tech]
```

Comment:
Maybe we could add more tags related to this article?

Reply:
Definitely. Already added in PR #72